### PR TITLE
blog - Using The RESTEasy Galleon Feature Pack In WildFly

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
@jamezp This is related with https://github.com/resteasy/resteasy-spring/releases/tag/2.0.0.Final could you please help to review it? Thanks!

File: _posts/2021-11-22-feature-pack.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+layout:     post
+title:      "Using The RESTEasy Galleon Feature Pack In WildFly"
+subtitle:   ""
+date:       Nov 22, 2021
+author:     Wei Nan Li
+---
+
+For WildFly/JBoss EAP integration, RESTEasy has migrated from using `jboss-modules` to the new Galleon Feature Pack scheme. Here are the relative modules:
```
Comment (jamezp) [inline _posts/2021-11-22-feature-pack.md:9]: We might want to re-word this a bit. We should say something like "RESTEasy has migrated from using a ZIP of jboss-modules" or something along those lines as WildFly still uses jboss-modules.

**Review by jamezp — COMMENTED** [review]
Just the one minor change @liweinan. Otherwise it looks great! One thing you could mention if you want too is to you should just need to execute:
```
galleon.sh install org.jboss.resteasy.spring:galleon-feature-pack:2.0.0.Final --dir=~/my-wildfly
```
The other two installs will work and there is nothing wrong with that, but they are not required.

**Comment by jamezp** [comment]
And one more thing. Should we mention that it installs a `org.springframework.spring` module too?

**Comment by liweinan** [comment]
@jamezp Thanks for reviewing! I'll do the modification.

**Pull request merged**